internal/parser/simple_parser_rules.go

```diff
@@ -50,6 +50,10 @@ func (p *simpleParser) parseSQLStatement(r reporter) (stmt *ast.SQLStmt) {
 		stmt.BeginStmt = p.parseBeginStmt(r)
 	case token.KeywordCommit:
 		stmt.CommitStmt = p.parseCommitStmt(r)
+	case token.KeywordCreate:
+		// for now we can directly add create index as the parsing function
+		// later there must be an intermediate layer to take care of different CREATE functions
+		stmt.CreateIndexStmt = p.parseCreateIndexStmt(r)
```

Comment:
Do this now.

Reply:
Sure. How would you do this? I will need to peek the next element?

Reply:
ParserStmt -> parseCreateStmt -> create index, table etc

Reply:
Done